src/_pytest/compat.py

```diff
@@ -308,13 +307,10 @@ def get_real_method(obj, holder):
 
 
 def getfslineno(obj) -> Tuple[Union[str, py.path.local], int]:
-    # xxx let decorators etc specify a sane ordering
-    obj = get_real_func(obj)
-    if hasattr(obj, "place_as"):
-        obj = obj.place_as
-    fslineno = _pytest._code.getfslineno(obj)
-    assert isinstance(fslineno[1], int), obj
-    return fslineno
+    """(**Deprecated**, use _pytest._code.source.getfslineno directly)"""
+    import _pytest._code.source
+
+    return _pytest._code.source.getfslineno(obj)
```

Comment:
Can't we remove this right away? Everything in `compat` is internal, so we are free to remove this function now.

Reply:
Sure.. :)